.github/workflows/ci.yml

```diff
@@ -29,3 +29,19 @@ jobs:
         env:
           CGO_ENABLED: 0
         run: go test -v ./...
+
+  test-install:
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        shell: [sh, bash, zsh]
```

Comment:
Rustup supports more shells, can we test a few more of them please?

Reply:
sure. I'm expecting that some of the tests will fail and we just proceed with merging until fixing the tests for other shells

Reply:
Make sure this is tracked before merging please.